### PR TITLE
Add SetColsVisibleFrom() to fix #559

### LINKED ISSUE #559: Hide all columns or rows, starting from one
Hello, I'd like to hide all rows or columns from one.

At this point, a function using the current implementation of changing the visibility of a row makes the file too large (and it takes time to do so..).

### PR DESCRIPTION
**Pull request by Amodio**
# PR Details

We need a function to hide both rows or columns from one, this PR deals only with columns.

## Description

Fixed some typos. Creation of SetColsVisibleFrom(), which is tested.

## Related Issue

#559 

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: col.go [hunk 1]
```diff
@@ -91,6 +91,46 @@ func (f *File) SetColVisible(sheet, col string, visible bool) error {
 	return err
 }
 
+// SetColsVisibleFrom provides a function to set visible of all the columns
+// after a given one in a worksheet. For example, hide column D in Sheet1:
+//
+//    err := f.SetColsVisibleFrom("Sheet1", "D", false)
+//
+func (f *File) SetColsVisibleFrom(sheet, col string, visible bool) error {
```
Comment (xuri) [inline col.go]: I propose that modify the exist function `SetColVisible` like `SetColStyle`:

```go
func SetColVisible(sheet, columns string, visible bool) error
```

to make it accept the specified column range:

```go
SetColVisible("Sheet1", "F", false)
SetColVisible("Sheet1", "F:XFD", false)
```

Comment (Amodio) [reply col.go]: Ok, i'll post that soon.

Comment (Amodio) [reply col.go]: Please see PR #565

**Comment by Amodio** [comment]
Reposted: PR #565

**Pull request closed without merging**